cmp/ag-grid/AgGrid.scss

```diff
@@ -180,10 +180,11 @@
     display: flex;
     align-items: center;
 
-    & > span {
+    & > * {
       flex: 1;
       overflow: hidden;
       text-overflow: ellipsis;
+      min-width: 0;
```

Comment:
It would also be good to comment this `.ag-cell-value` rule generally to lay out the goals of these styles - I have found myself coming back into stylesheets and not being sure why things are how they are, often I think because we tend to comment less but also because there seem to be fewer opportunities on the CSS side to make the code itself expressive.

In general I _think_ our goals here are to:
* Use `display: flex` on `.ag-cell-value` to support vertical centering of cell contents.
* Set `flex: 1` on the next nested element to ensure it takes up the entire cell width.
* Set the next three rules to ensure it clips and elides strings with ....  (Is `min-width` a trick to ensure that actually happens - something about putting the entire element into a "sized" mode where eliding could occur, even though we're clearly saying as little as possible about sizing?)